developer/src/kmc-ldml/test/test-compiler-e2e.ts

```diff
@@ -4,11 +4,13 @@ import hextobin from '@keymanapp/hextobin';
 import { KMXBuilder } from '@keymanapp/common-types';
 import {checkMessages, compileKeyboard, compilerTestCallbacks, compilerTestOptions, makePathToFixture} from './helpers/index.js';
 import { LdmlKeyboardCompiler } from '../src/compiler/compiler.js';
+import { CompilerMessages } from '../src/compiler/messages.js';
 
 describe('compiler-tests', function() {
   this.slow(500); // 0.5 sec -- json schema validation takes a while
 
   it('should-build-fixtures', async function() {
```

Comment:
```suggestion
  before(function() {
    compilerTestCallbacks.clear();
  });
  
  it('should-build-fixtures', async function() {
```

This is the pattern we are using elsewhere.

Reply:
much better.